### ISSUE TITLE
Admins Need a Round Info Panel

### ISSUE THREAD

**Issue by Geekyhobo** [space-wizards/space-station-14#24403]
## Description

SSL, Request was made in [discord](https://discord.com/channels/310555209753690112/1182777817105190982/1184705104658563102), The idea being that there would be a toggleable panel that just shows some basic round info, Shows crew manifest and round time so admins don't have to spawn a random PDA.

**Comment by Hmeister-fake** [comment]
Cant you already check all the players with the Ahelp menu?

**Comment by Geekyhobo** [comment]
> Cant you already check all the players with the Ahelp menu?

Yes, but specifically this request was for the crew manifest ui, so just names with jobs and job icons
